### PR TITLE
Add rule 'rails_migrations_pending'

### PR DESCRIPTION
**Pull request by abraham-chan**
Add rule to fix a common Rails issue when running tests. 

Rails source code that generates the message: https://github.com/rails/rails/blob/main/activerecord/lib/active_record/migration.rb#L152

Example output on my console:
```
abrahamchan [master] > bin/rspec     
Running via Spring preloader in process 35661
Migrations are pending. To resolve this issue, run:

        rails db:migrate RAILS_ENV=test
No examples found.

abrahamchan [master] > fuck
rails db:migrate RAILS_ENV=test [enter/↑/↓/ctrl+c]
== 20210627174953 Example: migrating ==========================================
== 20210627174953 Example: migrated (0.0000s) =================================

abrahamchan [master] > bin/rspec
[successful rspec output]
```

### CODE REVIEW

File: thefuck/rules/rails_migrations_pending.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import re
+
+
+SUGGESTION_REGEX = r"To resolve this issue, run:\s+(.*?)\n"
+
+
+def match(command):
+    return ("Migrations are pending. To resolve this issue, run:" in command.output)
```
Comment (scorphus) [inline thefuck/rules/rails_migrations_pending.py:8]:
```suggestion
    return "Migrations are pending. To resolve this issue, run:" in command.output
```

**Review by scorphus — APPROVED** [review]
Looks good. Thanks.

Please consider my sugestion below.

File: tests/rules/test_rails_migrations_pending.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+)
+def test_not_match(command):
+    assert not match(command)
+
+
+@pytest.mark.parametrize(
+    "command, new_command",
+    [
+        (Command("", output_env_development), "rails db:migrate RAILS_ENV=development"),
+        (Command("", output_env_test), "bin/rails db:migrate RAILS_ENV=test"),
```
Comment (scorphus) [inline tests/rules/test_rails_migrations_pending.py:42]: Reviewing it again, I thought that the corrected command could include the one initially intended by the user.

```suggestion
        (Command("bin/rspec", output_env_development), "rails db:migrate RAILS_ENV=development && bin/rspec"),
        (Command("bin/rspec", output_env_test), "bin/rails db:migrate RAILS_ENV=test && bin/rspec"),
```

Make sure to use `shell._and` to join the commands together.

File: thefuck/rules/rails_migrations_pending.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import re
+
+
+SUGGESTION_REGEX = r"To resolve this issue, run:\s+(.*?)\n"
+
+
+def match(command):
+    return "Migrations are pending. To resolve this issue, run:" in command.output
```
Comment (scorphus) [inline thefuck/rules/rails_migrations_pending.py:8]: Nit: should this rule be used only for `bin/rspec`?

```suggestion
@for_app("bin/rspec")
def match(command):
    return "Migrations are pending. To resolve this issue, run:" in command.output
```

**Review by scorphus — CHANGES_REQUESTED** [review]

File: thefuck/rules/rails_migrations_pending.py (hunk 3 above)
Comment (abraham-chan) [reply thefuck/rules/rails_migrations_pending.py:8]: No, I happen to use rspec but there are other testing libraries for Rails. Theoretically any command can trigger a `PendingMigrationError`

File: tests/rules/test_rails_migrations_pending.py (hunk 2 above)
Comment (abraham-chan) [reply tests/rules/test_rails_migrations_pending.py]: Thanks, done

**Review by scorphus — APPROVED** [review]
Good job, @abraham-chan! Thanks for contributing! :shipit:

**Pull request merged**